File: spring/web/SpringMvcTesting/src/main/java/com/instil/mvc/services/DeliveriesController.java
package com.instil.mvc.services;

import com.instil.mvc.model.Delivery;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.LinkedList;
import java.util.List;

@RestController
@RequestMapping("/deliveries")
public class DeliveriesController {
    @GetMapping(produces = "application/json")
    public List<Delivery> viewAllAsJson() {
        return new LinkedList<>();
    }
}

File: spring/web/SpringMvcTestingKotlin/src/main/kotlin/demos/spring/boot/rest/services/DeliveriesController.kt
package demos.spring.boot.rest.services

import demos.spring.boot.rest.model.Delivery
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/deliveries")
class DeliveriesController {
    @GetMapping(produces = ["application/json"])
    fun viewAllAsJson() = emptyList<Delivery>()
}
